### PR TITLE
fix(dev): upgrade env /2

### PR DESCRIPTION
**Pull request by bodinsamuel**
- Upgrade some others packages (runtime this time)
- Fix got upgrade and use keep alive agent 
- Reduce bootstrap amount (100 does not even pass on my computer)

### CODE REVIEW

File: src/utils/request.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import http from 'http';
+import https from 'https';
+
+import got from 'got';
+
+const options = {
+  keepAlive: true,
+  timeout: 60000,
```
Comment (Haroenv) [inline src/utils/request.js:8]:
```suggestion
  timeout: 60_000,
```

File: package.json [hunk 2]
```diff
@@ -23,21 +23,21 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@sentry/node": "5.30.0",
-    "algoliasearch": "3.35.1",
+    "@sentry/node": "6.5.1",
+    "algoliasearch": "4.9.1",
```
Comment (Haroenv) [inline package.json:27]: aren't there breaking changes in the algolia usage?

File: src/utils/request.js (hunk 1 above)
Comment (bodinsamuel) [reply src/utils/request.js:8]: this should be an eslint rule 😏

File: package.json (hunk 2 above)
Comment (bodinsamuel) [reply package.json:27]: well you are better suited to answer to that, but the basic client creation and save objects seems to work fine.
Note I tested locally with bootstrap with 100 Packages.

File: src/utils/request.js (hunk 1 above)
Comment (bodinsamuel) [reply src/utils/request.js]: actually eslint does not liky

File: package.json (hunk 2 above)
Comment (Haroenv) [reply package.json:27]: If it was typescript we could find more of the errors, but for example:

```
async function putDefaultSettings(index, config) {
  await index.setSettings(config.indexSettings);

  await index.batchSynonyms(config.indexSynonyms, {
    replaceExistingSynonyms: true,
  });
  const { taskID } = await index.batchRules(config.indexRules, {
    replaceExistingRules: true,
  });

  await index.waitTask(taskID);
}
```

must become:

```
  await index.setSettings(config.indexSettings);

  await index.saveSynonyms(config.indexSynonyms, {
    replaceExistingSynonyms: true,
  });
  const { taskID } = await index.saveRules(config.indexRules, {
    replaceExistingRules: true,
  });

  await index.waitTask(taskID);
}
```

Comment (bodinsamuel) [reply package.json:27]: ah good catch, this one is executed only if the original index does not exists. I missed it 👌🏻 
in the next PRs I'll try to create integration test, a lot of things are untested right now, too easy to make mistake (and I do a lot of them ahah)

Comment (bodinsamuel) [reply package.json:27]: Screen Shot 2021-06-04 at 17 21 09

**Pull request merged**